manifest_template.yml

```diff
@@ -97,21 +109,26 @@ apigee:
         environments: [ {{ ENV.name }} ]
         proxies:
           - e-referrals-service-api-{{ VARIANT.service | default(VARIANT.name) }}-{{ ENV.name }}
+          { % if ENV.name == 'dev' % }
+          - identity-service-dep-{{ ENV.name }}
+          - identity-service-mock-{{ ENV.name }}
```

Comment:
Very minor, but I guess given this proxy is being associated in both the `if` and `else` block it might be simpler for this proxy to be associated outside of the if statement above.